package.json

```diff
@@ -5,16 +5,18 @@
   "private": true,
   "scripts": {
     "lint": "eslint . --cache",
-    "postinstall": "npm run build && npm prune",
+    "postinstall": "npm prune",
     "prestart": "npm run start:mongo",
-    "start": "nodemon --ignore build/ -e js,jsx --exec \"npm run build && nf start\"; npm stop",
+    "start": "sh scripts/start.sh",
     "stop": "npm run stop:mongo",
     "start:mongo": "mongod --config mongod.conf --fork",
     "stop:mongo": "killall -9 mongod && echo Press ENTER to continue",
-    "build": "npm run build:clean && npm run build:server && npm run build:common",
     "build:clean": "rimraf build",
     "build:common": "babel --source-root src --source-maps inline -d build/common src/common",
     "build:server": "babel --source-root src --source-maps inline -d build/server src/server",
+    "build:client": "sh scripts/build-client.sh",
+    "build:dev": "npm run build:server && npm run build:client",
```

Comment:
Yeah, it's not set up correctly right now. I just took the lines from yours that will be used in the future